lighthouse-core/test/audits/byte-efficiency/unused-javascript-test.js

```diff
@@ -81,15 +96,20 @@ describe('UnusedJavaScript audit', () => {
       'https://www.google.com/inline.html',
     ]);
 
+    // Only two scripts should meet the unused bytes threshold.
+    expect(result.items).toHaveLength(2);
+
     const scriptBWaste = result.items[0];
+    assert.equal(scriptBWaste.url, `${domain}/scriptB.js`);
     assert.equal(scriptBWaste.totalBytes, 50000);
     assert.equal(scriptBWaste.wastedBytes, 12500);
     assert.equal(scriptBWaste.wastedPercent, 25);
 
-    const inlineWaste = result.items[1];
-    assert.equal(inlineWaste.totalBytes, 21000);
-    assert.equal(inlineWaste.wastedBytes, 6000);
-    assert.equal(Math.round(inlineWaste.wastedPercent), 29);
+    const inlineBWaste = result.items[1];
+    assert.equal(inlineBWaste.url, `${domain}/inline.html`);
+    assert.equal(inlineBWaste.totalBytes, 15000);
```

Comment:
Why does the total bytes change here?

Reply:
> > I've updated the PR to make fewer user-visible changes in the report (will do in follow up, simpler to review PRs).
> 
> sounds great. maybe can you give a bullet-list preview of what those changes/PR will be?

It's in the PR description.

Reply:
LOL @paulirish gh ui did something funky here.

Reply:
@adamraine previously the inline scripts were all combined into one. now they are separate, and only one of the two in this test are big enough for the audit to care about.